src/unix/pthread-fixes.c

```diff
@@ -39,19 +39,23 @@
 
 int uv__pthread_sigmask(int how, const sigset_t* set, sigset_t* oset) {
   static int workaround;
+  int errno;
```

Comment:
It's less confusing to human readers; if you glance at the code below, you could be mistaken thinking that the global errno gets reassigned.